lib/fizzy/execute.cpp

```diff
@@ -17,6 +17,28 @@ struct LabelContext
     size_t stack_height = 0;
 };
 
+void branch(uint32_t label_idx, Stack<LabelContext>& labels, Stack<uint64_t>& stack,
```

Comment:
I guess we could benchmark now whether this should be inlined or not.

Reply:
`mul256_opt0` uses both `br` and `br_if`.

Run both master and this a few times, there's some variance but they seem to be very close.